openmetadata-ui/src/main/resources/ui/src/components/TopicDetails/TopicSchema/TopicSchema.tsx

```diff
@@ -0,0 +1,335 @@
+              {t('label.no-description')}
+            </Typography.Text>
+          )}
+        </>
+        {isReadOnly && !hasDescriptionEditAccess ? null : (
+          <Button
+            className="p-0 opacity-0 group-hover-opacity-100"
+            data-testid="edit-button"
+            icon={
+              <SVGIcons
+                alt={t('label.edit')}
+                icon="icon-edit"
+                title={t('label.edit')}
+                width="16px"
+              />
+            }
+            type="text"
+            onClick={() => setEditFieldDescription(record)}
+          />
+        )}
+      </Space>
+    );
+  };
+
+  const renderFieldTags: CellRendered<Field, 'tags'> = (
+    tags,
+    record: Field
+  ) => {
+    const isSelectedField = editFieldTags?.name === record.name;
+    const styleFlag = isSelectedField || !isEmpty(tags);
+
+    return (
+      <>
+        {isReadOnly ? (
+          <Space wrap>
+            <TagsViewer sizeCap={-1} tags={tags || []} />
+          </Space>
+        ) : (
+          <Space
+            align={styleFlag ? 'start' : 'center'}
+            className="justify-between"
+            data-testid="tags-wrapper"
+            direction={styleFlag ? 'vertical' : 'horizontal'}
+            onClick={() => handleAddTagClick(record)}>
+            <TagsContainer
+              className="w-max-256"
+              editable={isSelectedField}
+              isLoading={isTagLoading && isSelectedField}
+              selectedTags={tags || []}
+              showAddTagButton={hasTagEditAccess}
+              size="small"
+              tagList={tagList}
+              type="label"
+              onCancel={() => setEditFieldTags(undefined)}
+              onSelectionChange={handleFieldTagsChange}
+            />
+          </Space>
+        )}
+      </>
+    );
+  };
+
+  const columns: ColumnsType<Field> = useMemo(
+    () => [
+      {
+        title: t('label.name'),
+        dataIndex: 'name',
+        key: 'name',
+        accessor: 'name',
+        ellipsis: true,
+        width: 220,
+        render: (_, record: Field) => (
+          <Popover
+            destroyTooltipOnHide
+            content={getEntityName(record)}
+            trigger="hover">
+            <Typography.Text>{getEntityName(record)}</Typography.Text>
+          </Popover>
+        ),
+      },
+      {
+        title: t('label.type'),
+        dataIndex: 'dataType',
+        key: 'dataType',
+        accessor: 'dataType',
+        ellipsis: true,
+        width: 220,
+        render: (dataType: Field['dataType']) => (
+          <Typography.Text>{dataType}</Typography.Text>
+        ),
+      },
+      {
+        title: t('label.description'),
+        dataIndex: 'description',
+        key: 'description',
+        accessor: 'description',
+        render: renderFieldDescription,
+      },
+      {
+        title: t('label.tag-plural'),
+        dataIndex: 'tags',
+        key: 'tags',
+        accessor: 'tags',
```

Comment:
What is an accessor? why do we need this?

Reply:
It's not needed, I will remove that.